craystack/codecs/multiset.py

```diff
@@ -0,0 +1,205 @@
+"""
+Copyright (c) Facebook, Inc. and its affiliates.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
```

Comment:
You might need to append the contents of https://github.com/facebookresearch/multiset-compression/blob/master/LICENSE to Craystack's LICENSE.